Model/Product/ProductInformationInterface.php

```diff
@@ -49,12 +49,17 @@ public function getShortDescription(): string;
 
     public function setShortDescription(string $shortDescription): self;
 
-    public function mapPublishProperties(self $draft): void;
+    public function mapPublishProperties(ProductInformationInterface $draft): void;
```

Comment:
why changing this here? self is used more than once - we should change it everywhere or no where :)

Reply:
my ide is marking this red

Reply:
but I will change it back then